### DISCUSSION TITLE
Comand Palette - Close with escape

### DISCUSSION THREAD

**Discussion by tstrijdhorst** [community/community#15622] [Other Feature Feedback, Questions, & Ideas]
I really like the command palette as it works similarly in other applications like Element/Slack etc. However for consistency it would be great if you could close the modal with escape.

(As a bonus you get points from vim users :grin:)

**Comment by BananaAcid** [comment]
Works for me already 🤷‍♂️

**Comment by doei** [comment]
Escape does not work for me either
